### PR TITLE
bugfix(CLI): if the user used default token and allow any IP to acces…

### PR DESCRIPTION
**Pull request by membphis**
…s Admin API,

  will show a WARNING message.

### What this PR does / why we need it:

if the user used the default token and allow any IP to access Admin API,   
will show a WARNING message.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: bin/apisix [hunk 1]
```diff
@@ -812,6 +814,51 @@ local function init()
...
+            end
+
+            if admin.key == "" then
+                io.stderr:write(help:format("ERROR: missing valid Admin API Key"), "\n")
+                os.exit(1)
+            end
+
+            if admin.key == "edd1c9f034335f136f87ad84b625c8f1" then
+                io.stderr:write(
+                    help:format("WARNING: using the default Key is very dangerous."),
```
Comment (moonming) [inline bin/apisix:855]: `using the default Key is very dangerous.` -> `using fixed API token has security risk, please modify "admin_key" in conf/config.yaml`

File: conf/config.yaml [hunk 2]
```diff
@@ -21,3 +21,9 @@
 #     host:
 #       - "http://127.0.0.1:2379"
 #
+apisix:
+  admin_key:
+    -
+      name: "admin"
+      key: edd1c9f034335f136f87ad84b625c8f1 # please update the default token for safe
```
Comment (moonming) [inline conf/config.yaml:28]: `using fixed API token has security risk, please update it when you deploy to production environment`

**Review by moonming — CHANGES_REQUESTED** [review]

File: bin/apisix [hunk 3]
```diff
@@ -799,6 +799,8 @@ version:    print the version of apisix
 ]])
 end
 
+
+local is_checked_admin_key
```
Comment (moonming) [inline bin/apisix:803]: `is_checked_admin_key` is confusing, please take a better name.

Comment (membphis) [reply bin/apisix:803]: `checked_admin_key`, any better name suggestions?

File: bin/apisix [hunk 4]
```diff
@@ -812,6 +814,52 @@ local function init()
...
+
+            if admin.key == "" then
+                io.stderr:write(help:format("ERROR: missing valid Admin API Key"), "\n")
+                os.exit(1)
+            end
+
+            if admin.key == "edd1c9f034335f136f87ad84b625c8f1" then
+                io.stderr:write(
+                    help:format([[WARNING: using fixed API token has security risk, ]]
+                                .. [[please modify "admin_key" in conf/config.yaml.]]),
```
Comment (moonming) [inline bin/apisix:856]: the same as https://github.com/apache/apisix/pull/2244/files#diff-62edadffc237f13dc28a694080d293a8R831

**Review by moonming — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**